crates/rpc/rpc-eth-api/src/helpers/fee.rs

```diff
@@ -166,7 +167,7 @@ pub trait EthFees: LoadFee {
                 for header in &headers {
                     base_fee_per_gas.push(header.base_fee_per_gas().unwrap_or_default() as u128);
                     gas_used_ratio.push(header.gas_used() as f64 / header.gas_limit() as f64);
-                    base_fee_per_blob_gas.push(header.blob_fee().unwrap_or_default());
```

Comment:
blob params have to be fork dependent here as well

Reply:
mentioned this in the comment above, this would be very annoying to do in this pr and is not critical